tools/shaderc/shaderc.cpp

```diff
@@ -719,10 +719,11 @@ namespace bgfx
 
 	struct Preprocessor
 	{
-		Preprocessor(const char* _filePath, bool _essl)
+		Preprocessor(const char* _filePath, bool _essl, bx::WriterI* _messageWriter)
```

Comment:
I moved all of _shaderWriter to _writer when it is the only writer parameter. Do you want the same for _messageWriter?

Reply:
`writef` is general function, not just for shader. In this case it's specific to messages.